core/src/test/kotlin/io/sharing/server/core/product/domain/ProductTest.kt

```diff
@@ -113,9 +134,10 @@ fun createProduct(
     licensePlate: String = "사와1234",
     status: ProductStatus = ProductStatus.AVAILABLE,
     region: Region = Region.GASAN,
-    description: String = "test"): Product {
-    return Product(
+    description: String = "test",
+    images: MutableList<String> = mutableListOf()): Product {
```

Comment:
```suggestion
    images: MutableList<String> = mutableListOf()
): Product {
```

이러는 편이 더 보기 좋고 일관성 있을거같아요